gravatar/src/main/java/com/gravatar/di/container/GravatarSdkContainer.kt

```diff
@@ -53,7 +56,7 @@ internal class GravatarSdkContainer private constructor() {
 
     fun getGravatarApiV3Service(okHttpClient: OkHttpClient? = null): GravatarApiService {
         return getRetrofitApiV3Builder().apply {
-            okHttpClient?.let { client(it) }
+            client(okHttpClient ?: OkHttpClient().newBuilder().addInterceptor(AuthenticationInterceptor()).build())
```

Comment:
We're not adding our interceptor to any dev-supplied HTTP client? Not necessarily saying we should, but since we made the AuthenticationInterceptor `internal`, devs will have to re-write it in order to support an API key...

Here's what this change would look like:

```suggestion
            client((okHttpClient ?: OkHttpClient()).newBuilder().addInterceptor(AuthenticationInterceptor()).build())
```

Reply:
It's a good point. Tbh, I had the same doubt. I've committed the suggested change.